### PR TITLE
Proposes a rubocop config to be used in all ruby projects

### PR DESCRIPTION
**Pull request by zamith**
Why:
- We want a way to make sure we are all following the same style guides

This change addresses the need by:
- Having a public place where it is easy to get the configs we all agree
  on, whenever we are starting a project, instead of always be copying and
  pasting from previous projects, which will ultimately lead to
  diferences.

### CODE REVIEW

File: linters/ruby/.rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  Enabled: false
+
+Style/SingleLineBlockParams:
+  Enabled: false
+
+Style/MultilineOperationIndentation:
+  EnforcedStyle: indented
+
+Metrics/MethodLength:
+  Enabled: false
```
Comment (naps62) [inline linters/ruby/.rubocop.yml]: what's the reason for disabling this last one?
I don't remember ever using it, and I think I actually prefer to have it

Comment (zamith) [inline linters/ruby/.rubocop.yml]: I think I had some situation where this was useful, but we can remove make
it true as a default.

**Comment by zamith** [comment]
@naps62 bump

**Comment by naps62** [comment]
There's just one more I'd like to propose, that I've been using in the past few weeks:

```
Style/TrailingComma:
  EnforcedStyleForMultiline: comma
```

this forces us to write:

```
{
  key: value1,
  key2: value2,
}
```

with that trailing comma at the last argument, which makes it easier to reorder stuff.

Add it if you agree. . Regardless of that, :shipit:

**Comment by zamith** [comment]
I have been thinking of adding that for a while now, Suspenders uses that and also Rust (I think). Let's go for it.

**Pull request merged**